src/components/EventCard.js

```diff
@@ -30,3 +36,94 @@ EventCard.propTypes = {
 }
 
 export default EventCard
+
+// styles
+
+export const EventCardLink = styled.a``
+
+const EventCardWrapper = styled.article`
+    label: event-card-wrapper;
+    color: ${p => {
+        if (p.isActive) return '#000'
+        return '#fff'
+    }};
+	position: relative;
+	padding: 15px 15px 30px;
+    background: ${p => {
+        if (p.isPast) return COLOURS.PURPLE_MEDIUM
+        if (p.isActive) return '#fff'
+
+        return 'none'
+    }};
+	border-radius: 5px;
+    border-style: solid;
+    border-width: 1px;
+    border-color: ${COLOURS.PURPLE_LIGHT};
+    border-color: ${p => {
+        if (p.isPast) return COLOURS.PURPLE_LIGHT
+        if (p.isActive) return '#fff'
+
+        return `${COLOURS.PURPLE_LIGHT}`
+    }};
+    min-height: 160px;
+	transition: background 0.4s ease-in-out;
+
+    
+    &:hover,
+    &:focus {
```

Comment:
I notice here :hover and :focus have & before them but they aren't there in src/components/Button.js, does that matter?